### PR TITLE
Add hint about how to use variables

### PR DESCRIPTION
**Pull request by timqian**
**Summary**

Failed to figure out in the docs about how to use the frontmatter variables. After trying `{{$frontmatter}}`, I guessed out it myself. It should save me an hour if the doc tells me how to invoke the variables

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [x] Code style update
- [ ] Refactor
- [ ] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [x] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: packages/docs/docs/guide/frontmatter.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ lang: en-US
 ---
 ```
 
-Between these triple-dashed lines, you can set predefined variables (see [below](#predefined-variables) for a reference), or even create custom ones of your own. These variables will then be available to you to access using <code>[$frontmatter](./global-computed.md#frontmatter)</code> at the rest of the page, plus all custom and theming components.
+Between these triple-dashed lines, you can set predefined variables (see [below](#predefined-variables) for a reference), or even create custom ones of your own. These variables will then be available to you to access using <code>{{[$frontmatter](./global-computed.md#frontmatter)}}</code> at the rest of the page, plus all custom and theming components.
```
Comment (bencodezen) [inline packages/docs/docs/guide/frontmatter.md:12]:
```suggestion
Between these triple-dashed lines, you can set predefined variables (see [below](#predefined-variables) for a reference), or even create custom ones of your own. These variables will then be available to you through the  <code>[$frontmatter](./global-computed.md#frontmatter)</code> variable.

Here's an example of how you could use it in your markdown file:

```
---
title: Blogging Like a Hacker
lang: en-US
---

# {{ $frontmatter.title }}

This blog post is written in the following language: {{ $frontmatter.lang }}.
```

**Comment by timqian** [comment]
@bencodezen Cool, this looks even better

File: packages/docs/docs/guide/frontmatter.md [hunk 2]
```diff
@@ -9,7 +9,10 @@ lang: en-US
 ---
 ```
 
-Between these triple-dashed lines, you can set predefined variables (see [below](#predefined-variables) for a reference), or even create custom ones of your own. These variables will then be available to you to access using <code>[$frontmatter](./global-computed.md#frontmatter)</code> at the rest of the page, plus all custom and theming components.
+Between these triple-dashed lines, you can set predefined variables (see [below](#predefined-variables) for a reference), or even create custom ones of your own. These variables will then be available to you through the  <code>[$frontmatter](./global-computed.md#frontmatter)</code> variable.
+
+Here's an example of how you could use it in your markdown file:
+
```
Comment (bencodezen) [inline packages/docs/docs/guide/frontmatter.md:15]:
```suggestion
````
---
title: My Blog Title
lang: English
---

# {{ $frontmatter.title }}

My blog post is written in {{ $frontmatter.language }}.

````


```

**Comment by bencodezen** [comment]
@timqian Glad to hear it! I'll go ahead and merge this while updating the example again locally since the suggestion feature is a little buggy with code blocks.

**Pull request merged**